src/libraries/System.Private.CoreLib/src/System/Enum.cs

```diff
@@ -934,7 +934,7 @@ private ulong ToUInt64()
                 case CorElementType.ELEMENT_TYPE_U1:
                     return data;
                 case CorElementType.ELEMENT_TYPE_BOOLEAN:
-                    return Convert.ToUInt64(Unsafe.As<byte, bool>(ref data), CultureInfo.InvariantCulture);
+                    return Convert.ToByte(Unsafe.As<byte, bool>(ref data));
```

Comment:
Could this be just:
```C#
return data != 0 ? 1 : 0;
```
?  That looks to be both the smallest IL and resulting asm.

Reply:
Sure, it's not what the rest of the class uses so was not sure about the preference

Reply:
Might be worth examining the rest of the uses to see if we could drop the dependency on Convert mostly or entirely.